packages/analytics-ma-browser/scripts/version/create-snippet-instructions.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
```

Comment:
perhaps all these build scripts should also be shared?

Reply:
Agree. Maybe move them into `analytics-client-common`?